linkcheck/tests/test_linkcheck.py

```diff
@@ -895,7 +898,7 @@ def test_checklinks_command(self):
             "1 internal URLs and 0 external URLs have been checked.\n"
         )
 
-        yesterday = datetime.now() - timedelta(days=1)
+        yesterday = timezone.now() - timedelta(days=1)
```

Comment:
This was just to silence an existing warning `RuntimeWarning: DateTimeField Url.last_checked received a naive datetime ... while time zone support is active.`